src/reducers/statusEffectsTick.js

```diff
@@ -2,21 +2,35 @@ import { TICK } from '../actions/tick';
 import { BLEED_OUT } from '../actions/bleeding';
 import { FEEL_TIRED } from '../actions/tired';
 
-// This came from the status effects file. TODO: abstract this somewhere else
-const isBleeding = effects => effects.find(effect => effect.type === BLEED_OUT) || { amount: 0 };
-const isTired = effects => effects.find(effect => effect.type === FEEL_TIRED) || { amount: 0 };
+// TODO: duped from stats
+const clamp = value => Math.max(Math.min(value, 100), 0);
```

Comment:
This is where the benefit of the `action => adjustment => new stats` architecture really starts to show; as far as status effects are concerned (at least with the basic ones we have so far), `TICK` is just a way of generating a list of stat adjustments to be applied, albeit in this case multiple adjustments for a single action.

The stats reducer is a single adjustment for a single action but 1) that stops being true if we change to actions being higher level in-game events and 2) single adjustment per action is just a special case of multiple adjustments per action.

Upshot is, rather than `statsWithEffectUpdate` you could basically turn most of your reducers so far into `updatesForAction` which then gets fed into `applyUpdatesToStats`.